In file included from /usr/include/dvdread/ifo_read.h:24,
                 from play_cell.c:34:
/usr/include/dvdread/ifo_types.h:32:2: error: #error "Must include <inttypes.h> or <stdint.h> before any libdvdread header."diff -Naur vamps-0.98.orig/play_cell/play_cell.c vamps-0.98/play_cell/play_cell.c
--- vamps-0.98.orig/play_cell/play_cell.c	2005-11-16 21:20:30.000000000 +0100
+++ vamps-0.98/play_cell/play_cell.c	2006-02-03 21:21:37.000000000 +0100
@@ -31,6 +31,7 @@
 #include <stdarg.h>
 #include <errno.h>
 #include <string.h>
+#include <inttypes.h>
 #include <dvdread/ifo_read.h>
 #include <dvdread/nav_read.h>
 
